docs/index.md

```diff
@@ -417,7 +417,13 @@ Knowing which companies are using this library is important to help prioritize t
 [**AppYourself**](https://appyourself.net){target="_blank" rel="nofollow"}
 { .card }
 
-[**Alma Media**](https://www.almamedia.fi/en/){target="_blank" rel="nofollow}
+[**Alma Media**](https://www.almamedia.fi/en/){target="_blank" rel="nofollow"}
+{ .card }
+
+[**Banxware**](https://www.banxware.com){target="_blank" rel="nofollow"}
+{ .card }
+
+[**Weschool**](https://www.weschool.com){target="_blank" rel="nofollow"}
```

Comment:
@dreamorosi Can you please make sure that the S is capitalised (We**S**chool) in the labels like our logo?

Thank you!

Reply:
Fixed in the last commit - my mistake.